source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -78,6 +78,7 @@ class FilterConfig {
         metadata_context_namespaces_(config.metadata_context_namespaces().begin(),
                                      config.metadata_context_namespaces().end()),
         include_peer_certificate_(config.include_peer_certificate()),
+        http_status_for_grpc_deny_(config.http_status_for_grpc_deny()),
```

Comment:
If this is indeed the canonical http status for grpc deny cases, I would add the word canonical in here and use the word 'on' instead of 'for' (precedent is `status_on_error`, IMO)

`canonicalHttpStatusOnGrpcDeny`